build.sbt

```diff
@@ -312,7 +313,7 @@ lazy val benchmarks212 = benchmarks(scala212)
 def project(id: String, path: String, publish: Boolean): Project = {
   Project(id, sbt.file(path))
     .settings(scalafmtOnCompile := true)
-    .settings(scalacOptions ++= Seq("-Xlint:unused", "-Ywarn-unused"))
+    .settings(scalacOptions ++= Seq("-Xlint:unused", "-Ywarn-unused", "-deprecation", "-Ywarn-macros:after"))
```

Comment:
`-Xfatal-warnings` FTW

Reply:
Ok, if I would be able to get rid of all warnings, then I will add this flag

Reply:
For now, I cannot add this flag, due to https://github.com/VirtusLab/ide-probe/issues/243